lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/src/main/java/io/lighty/modules/gnmi/simulatordevice/gnmi/GnmiCrudService.java

```diff
@@ -211,6 +211,10 @@ private Gnmi.UpdateResult processUpdateListNonSimpleValue(final Gnmi.Update upda
             resultingIdentifier = identifier.getParent();
         }
 
+        if (node.getIdentifier() instanceof AugmentationIdentifier && resultingIdentifier != null) {
```

Comment:
This is required due to:
https://github.com/PANTHEONtech/lighty/pull/971/files#diff-779afd3e1579082e1dab107038965cfefcefab7eeddc4d03f440ee0dfc517d18R211

And since .getParent() is annotated with Nullable, this will be reported in code smells.